node-feature-discovery-0.15.yaml

```diff
@@ -0,0 +1,94 @@
+package:
+  name: node-feature-discovery-0.15
+  version: 0.15.4
+  epoch: 0
+  description: Node feature discovery for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - protoc
+    provides:
+      - node-feature-discovery=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - protoc
+  environment:
+    GRPC_GO_LOG_SEVERITY_LEVEL: "INFO"
```

Comment:
This should be removed. It's set for the environment in the image. Setting this in the build environment won't do anything

Reply:
Removed it!